crates/wysiwyg/src/dom/nodes/container_node.rs

```diff
@@ -115,13 +115,14 @@ where
         }
     }
 
-    pub fn append_child(&mut self, mut child: DomNode<S>) {
+    pub fn append_child(&mut self, mut child: DomNode<S>) -> DomHandle {
```

Comment:
I also changed the return signatures of append, insert, etc. so they return the new `DomHandle`s and we don't have to re-calculate them or assume what they should be.